tests/test_distributed.py

```diff
@@ -25,6 +25,12 @@ def _test_init_parallel_context(parallel_context: ParallelContext):
     assert isinstance(parallel_context.world_rank_matrix, np.ndarray)
     assert isinstance(parallel_context.world_ranks_to_pg, dict)
 
+    local_rank = tuple(i.item() for i in np.where(parallel_context.world_rank_matrix == world_rank))
+    global_rank = parallel_context.get_global_rank(*local_rank)
+    assert isinstance(global_rank, np.int64), f"The type of global_rank is {type(global_rank)}"
+
+    assert global_rank == parallel_context.world_rank_matrix[local_rank]
```

Comment:
global_rank should equal to `dist.get_rank()`

Reply:
@xrsrke Thank you for the feedback! I hope I understood what was asked correctly and implemented the new change request :)